python/dgl/graphbolt/impl/sampled_subgraph_impl.py

```diff
@@ -68,6 +68,63 @@ def __post_init__(self):
             ), "Nodes in pairs should be of type torch.Tensor."
 
 
+def _to_reverse_ids(node_pair, reverse_row_node_ids, reverse_column_node_ids):
+    u, v = node_pair
+    if reverse_row_node_ids is not None:
+        u = reverse_row_node_ids[u]
+    if reverse_column_node_ids is not None:
+        v = reverse_column_node_ids[v]
+    return (u, v)
+
+
+def _relabel_homo_edges(lhs_edges, rhs_edges):
```

Comment:
Why do we have two sets of edges during relabeling?

Reply:
According to previous discussion, some sampling algorithm natually have column compressed from a view of "columnwise pick", but row is not compressed.

Reply:
This to to map the node ids of two set of edges to a consecutive range from 0. I concatenate them in this function.

Reply:
I see.  I would suggest (1) rename `lhs_edges` and `rhs_edges` to more informative names like `node_id_pairs`, `node_id_pairs_to_exclude`.

Reply:
Good point. I use `edges` and `edges_to_exclude` now.